acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/site/SiteSteps.java

```diff
@@ -79,6 +79,17 @@ public void iSeeLink(String linkTitle) throws Throwable {
         assertNotNull("Can see element", sitePage.findElementWithTitle(linkTitle));
     }
 
+    @Then("^I (?:can )?see element with data-uipath \"([^\"]+)\" with content \"([^\"]+)\"$")
+    public void iSeeElementWithUiPathAndContent(String uiPath, String content) throws Throwable {
+        assertNotNull("Can see element with data-uipath", sitePage.findElementWithUiPath(uiPath));
+        assertTrue(sitePage.findElementWithUiPath(uiPath).getText().contains(content));
```

Comment:
Can you use assertContainsString instead? it gi you more debug logging if this assert fails

Reply:
That method doesn't exist as far as I can see but I have changed it now